contrib/IECoreAlembic/include/IECoreAlembic/IGeomParamTraits.h

```diff
@@ -90,6 +90,8 @@ IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IN2dGeomParam, IECore
 IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IN3fGeomParam, IECore::V3fVectorData, IECore::GeometricData::Normal )
 IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IN3dGeomParam, IECore::V3dVectorData, IECore::GeometricData::Normal )
 
+IECOREALEMBIC_SPECIALISEGEOMPARAMTRAITS( Alembic::AbcGeom::IBoolGeomParam, IECore::IntVectorData, IECore::GeometricData::Numeric )
```

Comment:
Hmm, I really think we should be using bools for this so that we can round-trip data correctly when we add an Alembic writer, and so that renderers that do support bools directly (Arnold for instance) can be properly supported. Do you think you could either use a DataPromoteOp somewhere in your pipeline on the way into RenderMan, or sit on this as a local change until we add a replacement RenderMan backend (it's due an overhaul since Gaffer added a new renderer API more suited to threading and IPR).